torchci/pages/kpis.tsx

```diff
@@ -7,9 +7,10 @@ import { useState } from "react";
 const ROW_HEIGHT = 340;
 
 export default function Kpis() {
-    const [startTime, setStartTime] = useState(dayjs().startOf("year"));
+    // Start looking at data from 6 weeks in to avoid outlier data from the start of the year
```

Comment:
feel like this is a bit weird. We had data back then i thought.

Reply:
We do have data, but it's pretty spiky (e.g. high error rates).  That data is old enough that it doesn't provide much value to us anymore.

We soon might even want to change this start time to be a rolling window of "6 months ago"